wordpress-mobile/WordPress-iOS
# Issue/3970 implement blog settings

#3970 

First stab at implementing Blog settings on the app.

This PR just implements Site Title and Site Tagline.

How To Test:

* Open Blog Details
* Open Settings
* Change the Site Title or Tagline
* Press Save
* See if the change is reflected on the server

This should be tested on Sites hosted on WP.com and Self hosted sites with and without Jetpack.

Needs Review: @koke
